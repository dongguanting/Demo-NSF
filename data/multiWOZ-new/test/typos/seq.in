i'd like 2 find a train that leaves on monday and arrives by 7pm
i need to get away from peterborough
just 1 , pls .
leave stansted air aftr 11:45
id like 2 deprt from stevenage after 19:45
pls confirm when the 2sday train leaves
well . i am willing to go from cabridge to london liverpool street
i wud like 2 leave after 20:15
im lking fa train going to cambridge .
well its broxbourne on tuesday
id juss lyk to arrive by 1645 plz
yup , one tc plz
i wanna go to cambridge by train , i have to get there at 11:30
im departin frm ely
im headed 2 leicster frm cambridge on sat
oh . sorry its not morning 8:45 . its evening 8:45 .
i want to arrive by 19:30 please
train departing from london kings cross ?
saturday , to cambridge , pls
the train shud go 2 cambcambr en arrive by 14:00
plz book dat for 5 ppl
im travelin on mon and need 2 leave aftr 1630
i need to arrive by 11:15 .
i need a train that arrves at 12:30 on wed .
im departing frm birmingham new st. to cambridge
yes , i've 2 be in cambridge 12:30 , pls .
id like 2 go on sunday
i need a train leavin cmabridge after 17:30 .
2 stansted airprt on sat
looking for a train dep. mon. out of cam.
ya i ned three seats on the train plz .
iam looking for a tran that goes to london liverpool street and leafes after 17:15 .
this fri
i am lookinge four an train that departs form cambridge on sthursdai
is there any thing else i can help you with
i'll be lving frm camb.
birmingham new st
mday is wen i plan to trvl
just one piece of paper please .
ima go from liverpol st in lundon .
well . it is wednesday
i wuld like to find a train livin cambridge on monday
can u get my train from cam this sun ?
i needs to find trein that gonna arrive in stevenage by 11:45 .
trains leeving thu stansteeed daytim
im lking fa train lving from norwich .
pls help me get 2 cambs
i will b departing from norwich on wednesday
4 more people are going with me , book it
trains from cams 2 stansted ta!
well its 13:45 sunday
i only need 1 2day
actually the train i need is from cambridge to bishop stortford and not the other way round .
one is enough . please book
i'm gonna london king cross and i have to arrive before 16:45
find me please any train to cambridge
i'm lkng for train frm london on friday after 20:00
looking for train from cambridge
pls tell the train details of train from cambridge on sunday
plz verify it will b @ stevenage by 9 ?
i ned a train departing cfrom cabraineg
will be goin to london liverpool street on sunday .
i wanna take a train to cambridge
does train depart frm broxbourne station
sun
well . it would be better if leave after 13:45 from the station
can i plz get info for a train going from cambridge 2 broxbourne ?
1st train after 19:45 - id and $ ?
i'm gonna to stansted airport by train from cambridge
i ned to go 2 cambridg bi train
kindly , make it possible for me to depart at 8.00 to cambridge .
that is fine , may i purchase 6 tickets ?
please find me cambridge - norwich train
id like to go on sunday .
i must arrive no later than 12:30
i only want 1 pls
yes of course . it is sunday
train at 17:15 and later , please
book for 2 people
want to depart from cambridge on monday by 12:45 .
which train will b bakk by 9:15
can you tell me where iam . iam leaving from cambridge
i am looking 4 a train going to birmingham new street and shod leave on thurs
well its from cambridge . and i should arrive there by 17:15
of course . its is from cambridge to london liverpool
well it would be better if i arrive at 15:45 on tuesday at cambridge
i'm looking 4 train out of cambridge sat .
i want 2 go 2 kingz lyn
i need a to find a trai deprting from sansted airpor going too cambirdge
i'm leaving this saturday for cambridge
pls find my ely train
4 thurs .
all i neeed is too arrve by 14:15 .
ill be gng frm cambrige to broxborn
this sat
can you tell by which train i can leave at 08:30
yes that would be fine please book for 6 people .
pls help me get train 2 norwich
the train should leave arter 17:00 and should go 2 cambridge
well i need reservation for 4 people for that train
ely 1709 friday
train from cambs after 9 ta
i would like to bokk a rtarin for monday departing from kikngs lynn and arrivbing at famabridge at 1445
well . may i know its arrival time
i am looking for a locomotive to london kings cross that departs after 08:15 .
i need a train no earlier 20:45 from cambridge
london , liverpool street area
i would like 2 leave on thurs
i need to take a train on wednesday that leaves after 1600
i'm gonna go to kings lynn
it would be great if the train arrive by 9:45
i wood lyk to leave on sudnay
birmingham , new street , wednesday , please
i need to get a train out of cambridge
da train should go to stansted airport nd should arrive by 21:30 .
i wuld like 2 leav tues
whichever train goes 2 stansted airportt on tuesday from cabridge arriving close to 21:30 .
great can i book that for 8 people and get a refrence number pealse ?
lookn 4 train
im lookin tgt london kings cross departing from cambridge
user : i am looking for a train. agent : i'd be happy to help you with that. can you tell me your departure location and where you are headed to? user : i am looking to go to london kings cross and depart from cambridge. agent : there are 70 trains making that trip, do you have a day or time ?
after 12:30 from broxbourne to cambridge
i am lkng 4 a train to broxbourne after 15:15 .
pls find train at 15.15 on sunday m
tuesday
need to go to cambridge frm stevenage , after 1045
wednesday
id like a train to london liverpool st that leaves after 10
ill b leaving from cambridge fri
which train is leaving from broxbournea
i would like head to cambridge
no i should arrive there by 17:15
im leaving wedesday .
does dat train arrive b4 17:15
need a train to cambridge
ely this fri
i leave saturday am at 8:45 .
schedule dictates leave cambridge on thursday at 08:45 and i wonder if you can issue a ticket for that day and time
i need 2 leave after 2015
just 1
nrwch 2 cmbrdg
omg arrive in cambridge @ 14:15
well there are two only
please provide me information about the trains to cambridge that leaves on friday
im looking for aa train from cambridge
i have a fliht to mexico from standed airport , can you tell me hw i would ger there ?
id like too trvl in thrsday .
i cant leave til after 16:15 .
i'm looking for reservations for two people on the train on tuesday at 10:30 with final destination broxbourne and leaving cambridge .
i am reaching their on monday at 14:15
i wuld like to trvl on mon after 12:00 .
i want 2 go 2 licester
nned to go to cambrid monday
just 1 pls .
i to bok a train that departs from cambridge and arrives in peterbrough by 21:45 .
i ned a tran frum london loverpool streete station on wednasday .
i ned to arive bye 14:45 .
i want 2 leave on weds
y that works great can i book that for 5 people please ?
cn i fnd a train to peterborough departing frm cambridge
im going 2 kings lyn
i'ams woood have leavd frum cambrige
the train sd leave after 10:15 + go 2 peterborogh
train from da cambridge station to da ely station .
am looking 4 a train it should arrive by 1830 and should go 2 peterborough
i need 2 leave on sun from cambride
im looking for train going to cambride pls
i'm lookn 4 a train going 2 cambridge
user : i will be leaving from brookshite .
plz get me from london liverpool street to cambridge asap . thnks
im gonna need 1 ticket
cold you tel me when the tran arives in london liverpool stret ?
i actualy need 2 leave on sat
i nd a train t thday leaving from cambridge t ely .
im leavin @ 545 , after that is cool
i'm am loking for a trainn from campbridge to leicester .
i will be leavin on monday .
i'd lik to fiind a train to peteborough
can u reserve 6 tix onteh next trai nafter 1155 ?
i wuold liek to leve after 9:00 on snday
im goin to cambridge .
i'll willl be leavin from london liverpool streeet
night train after 9 , start at liverpool st
pls , i need to borard the train that 'll leave cmbdge and arrive in k. l. at 8am .
wanna get a train 2 cambridge plz
leavin sun need to get thur by 13:00
im loking for a train ging to broxbourne pleas .
i'm loking for a trane that will be gong to london kings cross on wed .
i need to arive by 18:00 .
pls find a train that deaprts from peterborogh + arrives i ncambridge by 845
i wold like to leav on sat from lond0n liverpool stret
what'e the sitch on trains from cambridge to ely ?
book for 3 and a reference nmbr pls ?
ineed 2 bok a train
i'm at biishops strotford and would like 2 go 2 cambridge .
i wd like to travel this weds n needto leave after 815
user : would you be able to find me a train leaving on sunday ?
wednesday is the plan atm
i readlly must arive by 18:15 to mkae my meting .
need a train to take 5me to cambridge
i wud lke to lve frm stevenage on thrsdy and get to cambridge by 9:15
ima leave leicester n need to leave by 20:30
i ned a train on tuesday out of kings lynn
i wanna leave on tues .
train on tues at 930 from kings to cam
i need a trane out 2 london lverpool st. on saturday .
looking to leave after 15:30 frm cambridge
i'm goin 2 cambridge
i'll be goign to perterborough from cambrigde .
yes pls bok it for 1 person and provide the refer numbr
pls tell me abt train leaving after 11:45
ill be leaving stansteed airport fri 2 head twds cambridge
user : that would work for me . can i get a train for 7 people ?
the departing trane should leve after 1415 , headed 2 burmingham new st
im loking for a train that goes 2 cambidge aruond 10:45
you need to go by train from cambrige to noeich .
are there any trains leaving form cambride on thu ?
im heading 2 broxborn
pls im looking the one that arrive at 3pm
im gonna hav 6 pls
i neid a tran to peteroborgh from cambrige on sundai .
i wd like 2 leve after 1815 pls
iâ€™m gonna deport from norwich
i wd like 2 leve atfer 14:03
i ned 2 book a train deprating from cabmridge after 915
i am lookin fr a train departing frm london liverpool street on thursday to go to cambridge .
i have to board a train to cambridge .
no , please confirm the train time comes from kings lynn .
i wld like to get there on thur
id like togo after 1000
im looking 4 a train
i ned too leave thrusday frm cambredge .
trns that r ging to ptrbrgh
i am looeking to go frim cambridg to peterborgh and leaf aftir 14:15
i'll b trvlling on 2sday
i need a rain going 4rm stevenage 2 cambrige
yeah , you got any on teusday and arriving by 9:00 ?
i wanted to leaf for cambridgeby train on wendsay
i'll b coming frm london k.c.
need train frm london 2 kings cross
cambridge .
user : i'm leaving london liverpool street on sunday after 20:30 .
isnt there smth closer to a 20:30 departure ?
i'm ready to leave sunday .
yes , can i get a tckt for 8 ppl , pls ?
gimme two
does that train leave on fri ?
tx you .
i ned train 2 ely fr camebrdge plz
id like 2 leave on fri n arrive by 1445
stevenage; cambridge .
yes i wold like to bok it for 8 people and i wil ned the refer numbr .
i wnat 2 leave form stevenage on sat
i wud lke 2 leav after 12:15
i wll depart aby tim as long as the trian arrives by 20:45
i need 2 make a train reservatoin for fri n i need it to arriveby 1900
i wil be daparting from cambridge .
i will be gong 2 stanstd airport
to cambridge and leavin london liver pool street
user : i need a train to take me from cambridge to broxbourne .
i wll be travlling on onday
lemme get a train from cambridge to bishops .
thur , after 20:15 .
i need a trian from cambrige after 9 thrusday thx
iam loking for a train 2 cambridge on sat
yesss saturday pls
i'd lick to arrival at 13:45 .
i ned a train that leves on wed with an arival of 12:30 .
i'm travelinga frum camebridge to the londan liverpoole streete .
user : i'd like to find a train tuesday departing from leicester .
i wd like 2 leave after 1115
yes looking 2 go from leicester 2 cambridge aftr 11:15
id like 2 find a train from leicesster on sat
i'm gonna to cambrige and need to go after 12
i am looking for a train from london kings cross and leaves on wednesday
i need a train lving frm london liverpool strt on fri .
i ned to arive by 41:00
user: excuse me , i wanna a train to cambridge , help pls ?
i wold lik to leave on tueday and arive by 13:30 .
user : i am looking for a train on friday that leaves after 21:45 .
i'll b lvng frm cambridge nd i wnt to go to stansted airport
in eed 2 leave after 2145
im deporting london kongâ€™s cross after 19:00 .
i ned it for 6 peopel .
im looking 4 a train leaving london kings corss + arriving by 14:30
im gong 2 cambrdge
i'll take a booking 4 seven ppl at 10 .
i wili ned 2 keave from cambridge
i'd like 2 travel on a thursday .
get ther by 17:45 pls
i wud like to catch a train leaving cambridge on sunday .
heading too leicester plz
gr8 , i want to leave after 10:15 , ok ?
im looking for a train htat leaves biships stortford and goesto cambride
saturday pls
'wood like t arrive in cmbridge by 8:15
i ned a train from lodon kings cros to cabridge .
is thr rian schdle for sudy
im looking 4 trn lving from stevenage
i need to get 2 cambrige this fri .
wan2 to go a little after 13:30 .