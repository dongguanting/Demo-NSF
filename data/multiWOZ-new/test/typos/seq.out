O O O O O O O O O B-day O O O B-arrive
O O O O O O B-depart
O B-people O O O
O B-depart I-depart O B-leave
O O O O O B-depart O B-leave
O O O O B-day O O
O O O O O O O O O O B-depart I-depart I-depart
O O O O O O B-leave
O O O O O O B-dest O
O O B-depart O B-day
O O O O O O B-arrive O
O O B-people O O
O O O O B-dest O O O O O O O O O B-arrive
O O O B-depart
O O O B-dest O B-depart O B-day
O O O O O O O O O O O O
O O O O O B-arrive O
O O O B-depart I-depart I-depart O
B-day O O B-dest O O
O O O O O B-dest O O O B-arrive
O O O O B-people O
O O O B-day O O O O O B-leave
O O O O O B-arrive O
O O O O O O O B-arrive O B-day O
O O O B-depart I-depart I-depart O B-dest
O O O O O O B-dest B-arrive O O O
O O O O O B-day
O O O O O B-depart O B-leave O
O B-dest I-dest O B-day
O O O O O B-day O O B-depart
O O O B-people O O O O O O
O O O O O O O O B-dest I-dest I-dest O O O B-leave O
O B-day
O O O O O O O O O B-depart O B-day
O O O O O O O O O O
O O O O B-depart
B-dest I-dest I-dest
B-day O O O O O O
O B-people O O O O O
O O O B-depart I-depart O I-depart O
O O O O B-day
O O O O O O O O B-depart O B-day
O O O O O O B-depart O B-day O
O O O O O O O O O B-dest O B-arrive O
O O B-day B-dest O
O O O O O O B-depart O
O O O O O B-dest
O O O O O B-depart O B-day
B-people O O O O O O O O O
O O B-depart O B-dest O
O O B-leave O
O O O B-people O
O O O O O O O B-depart O B-dest I-dest O O O O O O O
B-people O O O O O
O O B-dest I-dest I-dest O O O O O O B-arrive
O O O O O O B-dest
O O O O O B-depart O B-day O B-leave
O O O O B-depart
O O O O O O O O B-depart O B-day
O O O O O O B-dest O O O
O O O O O O B-depart
O O O O B-dest I-dest I-dest O O O
O O O O O O B-dest
O O O O B-depart O
B-day
O O O O O O O O B-leave I-leave O O O
O O O O O O O O O O B-depart O B-dest O
O O O B-leave O O O O O
O O O B-dest I-dest O O O B-depart
O O O O O B-dest O O
O O O O O O O O O O B-leave O B-dest O
O O O O O O O B-people O O
O O O B-depart O B-dest O
O O O O O B-day O
O O O O O O B-arrive
O O O B-people O
O O O O O O B-day
O O B-leave O O O O
B-people B-people B-people B-people
O O O O B-depart O B-day O B-arrive O
O O O O O O B-arrive
O O O O O O O O O O B-depart
O O O O O O O O B-dest I-dest I-dest O O O O B-day
O O O B-depart O O O O O O O B-arrive
O O O O O O B-depart O O O
O O O O O O O O O O O B-day O O
O O O O O O B-depart B-day O
O O O O O B-dest I-dest
O O O O O O O O O B-depart I-depart O O B-dest
O O O B-day O B-dest
O O O B-dest O
O B-day O
O O O O O O O B-arrive O
O O O O B-depart O B-dest
O B-day
O O O O O O O O O O B-leave
O O O O O O O O B-people O O
O O O O O O B-depart
O O O O O B-leave O O O O B-dest
O O O O O B-people O O O O
O B-arrive B-day
O O B-depart O O O
O O O O O O O O B-day O O B-depart I-depart O O O B-dest O B-arrive
O O O O O O O O
O O O O O O O B-dest I-dest I-dest O O O B-leave O
O O O O O O B-leave O B-depart
B-dest O I-dest I-dest O
O O O O O O B-day
O O O O O O O B-day O O B-leave I-leave
O O O O B-dest I-dest
O O O O O O O O O B-arrive
O O O O O O B-day
B-dest O I-dest I-dest O B-day O O
O O O O O O O O B-depart
O O O O O B-dest I-dest O O O O B-arrive O
O O O O O B-day
O O O O B-dest I-dest O B-day O O O O O B-arrive O
O O O O O O B-people O O O O O O O O
O O B-depart
O O O B-dest I-dest I-dest O O B-depart
O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O
O B-leave O B-depart O O
O O O O O O O B-dest O B-leave O
O O O O O O B-day O
B-day
O O O O B-dest O B-depart O O B-leave
B-day
O O O O O B-dest I-dest I-dest O O O B-leave
O O O O B-depart B-day
O O O O O B-depart
O O O O O B-dest
O O O O O O B-arrive
O O B-day O
O O O O O B-arrive
O O O O B-dest
B-depart O B-day
O O B-day O O B-leave O
O O O O O B-day O O O O O O O O O O O O O O O O
O O O O O B-leave
O B-people
B-depart O B-dest
O O O O O B-arrive
O O O B-people O
O O O O O O O O B-dest O O O B-day
O O O O O O B-depart
O O O O O O O B-dest I-dest O O O O O O O O O O O
O O O O O B-day O
O O O O O B-leave O
O O O O O B-people O O O O O B-day O B-arrive O O O B-dest O O B-depart O
O O O O O B-day O B-arrive
O O O O O O B-day O B-leave O
O O O O O B-dest
O O O O B-dest B-day
O B-people O O
O O O O O O O O B-depart O O O B-dest O B-arrive O
O O O O O B-depart I-depart I-depart O O B-day O
O O O O O B-arrive O
O O O O O B-day
O O O O O O O O O B-people O O O
O O O O O O B-dest O O B-depart
O O O O B-dest
O O O O O B-depart
O O O O O B-leave O O O B-dest
O O O B-depart O O O B-dest O O
O O O O O O O O O B-arrive O O O O B-dest
O O O O O B-day O B-depart
O O O O O O B-dest O
O O O O O O O B-dest
O O O O O O O B-depart O
O O O O B-depart I-depart I-depart O B-dest O O O
O O O B-people O
O O O O O O O O O B-dest I-dest B-dest O
O O O O O O B-day
O O O O O B-day O O B-depart O B-dest O
O O O B-leave O O O O O
O O O O O O O B-depart O B-dest O
O O O O O B-day O
O O O O O O O B-dest
O O O B-people O O O O O B-leave O
O O O O O B-leave I-leave O B-day
O O O B-dest O
O O O O O B-depart I-depart I-depart
O O O B-leave O O O O O
O O O O O O O O O O O B-depart O O O B-dest B-dest O O O
O O O O O B-depart O
O O O O O B-day O B-arrive
O O O O O O O B-dest O O
O O O O O O O O O O B-dest I-dest I-dest O B-day O
O O O O O B-arrive O
O O O O O O O B-depart O O O B-dest O B-arrive
O O O O O O B-day O O O O
O O O O O O B-depart O B-dest O
O O B-people O O O O O O
O O O O B-dest
O O B-depart I-depart O O O O O O B-dest O
O O O O O O B-day O O O O B-leave
O O O O O O O O O O O O O B-day O
B-day O O O O
O O O O O B-arrive O O O O O
O O O O O O O B-dest
O O O O O O B-depart O B-day O O O O O O
O O B-depart O O O O O B-leave
O O O O O B-day O O B-depart I-depart
O O O O B-day O
O O B-day O O O B-depart O B-dest
O O O O O O B-dest I-dest I-dest O B-day O
O O O O B-leave O B-depart
O O O B-dest
O O O O O B-dest O B-depart
O O O O O B-people O O O O O O
O O O O O O O B-leave
O O O B-depart I-depart B-day O O O B-dest
O O O O O O O O O O O O O O B-people O O
O O O O O O B-leave O O O B-dest I-dest I-dest
O O O O O O O O B-dest O B-arrive
O O O O O O O B-depart O B-dest O
O O O O O O B-depart O B-day O
O O O B-dest
O O O O O O O O B-arrive
O O O B-people O
O O O O O B-dest O B-depart O B-day O
O O O O O O B-leave O
O O O O B-depart
O O O O O O B-leave
O O O O O O O O B-depart O O
O O O O O O O O B-depart I-depart I-depart O B-day O O O B-dest O
O O O O O O O B-dest O
O O O O O O O O O B-depart I-depart O
O O O O O O O B-day
O O O O B-leave
O O O O B-depart
O O O O B-day O B-depart O
O O O O O B-dest
O O O O O O B-depart O O O O O B-leave
O O O O B-day
O O O O O O B-depart O B-dest
O O O O O O B-day O O O B-arrive O
O O O O O B-dest O O B-day
O O O O B-depart I-depart
O O O B-depart O I-depart O
B-dest O
O O O O B-depart I-depart I-depart O B-day O B-leave O
O O O O O O B-leave O O
O O O O B-day O
O O O O O O O O B-people O O O O
O B-people
O O O O O B-day O
O B-dest O
O O O O B-depart O O O
O O O O O B-day O O O B-arrive
B-depart B-dest O
O O O O O O O O B-people O O O O O O O O O
O O O O O B-depart O B-day
O O O O O O B-leave
O O O B-leave O O O O O O O O B-arrive
O O O O O O O O B-day O O O O O O B-arrive
O O O O O B-depart O
O O O O O B-dest I-dest
O B-dest O O B-depart I-depart I-depart I-depart
O O O O O O O O O O B-depart O B-dest O
O O O O O B-day
O O O O O B-depart O B-dest O
B-day O O B-leave O
O O O O O B-depart O B-leave B-day O
O O O O O O B-dest O B-day
O B-day O
O O O O O B-arrive O
O O O O O O O B-day O O O O B-arrive O
O O O B-depart O O B-dest I-dest I-dest O
O O O O O O O O B-day O O B-depart O
O O O O O O B-leave
O O O O O O O B-dest O O
O O O O O O O O O B-day
O O O B-dest O O O O O B-leave
O O O O O O O B-depart I-depart I-depart O O O B-day
O O O O O O B-depart I-depart I-depart O B-day O
O O O O O B-arrive
O O O O O O O O O B-dest O O O O
O O O O O O B-day O O O B-arrive O
O O O O O O O O O B-day O O O B-leave O
O O O O B-depart O O O O O O B-dest I-dest
O O O O O B-leave
O O B-depart I-depart I-depart O B-leave O
O O O O B-people O O
O O O O O O O B-depart I-depart I-depart O O B-arrive
O O O B-dest
O O O O O B-people O O B-leave O
O O O O O O B-depart
O O O O O O B-day O
O O O B-arrive O
O O O O O O O O B-depart O B-day O
O O B-dest O
O O O O O O O B-leave O O O
O O O O O O O B-depart I-depart O O B-dest
B-day O
O O O O O O O B-arrive
O O O O O B-depart I-depart I-depart O B-dest O
O O O O O B-day
O O O O O O B-depart
O O O O O B-dest O B-day O
O O O O O O B-leave O
